package.json

```diff
@@ -6,24 +6,22 @@
   "author": "Egbert Teeselink",
   "license": "MIT",
   "type": "module",
-  "source": "src/index.ts",
   "exports": {
-    "require": "./dist/index.cjs",
-    "default": "./dist/index.modern.js"
+    ".": {
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.cjs"
+    }
   },
   "main": "./dist/index.cjs",
-  "module": "./dist/index.module.js",
-  "unpkg": "./dist/index.umd.js",
   "types": "./dist/index.d.ts",
   "files": [
     "dist"
   ],
   "scripts": {
-    "build": "microbundle",
-    "dev": "microbundle watch",
```

Comment:
microbundle's watch mode is handy while developing - it recompiles to dist every time a source file changes. does unbuild have something similar?

Reply:
Unbuild doesn't have an option for it, but it would be possible using something like Nodemon:
```
"dev": "nodemon -w src -x 'npm run build'"
```

Reply:
Sounds fine! Would you be up for setting this up? I think we can merge the PR then. If other people run into trouble we'll re-adjust then :)

Reply:
I think it's ready to be merged. The author of the other PR also tested it.

Reply:
love it! thanks for your efforts :-)